js/tests/unit/dom/event-handler.spec.js

```diff
@@ -441,4 +441,28 @@ describe('EventHandler', () => {
       expect(i).toEqual(5)
     })
   })
+
+  describe('general functionality', () => {
+    it('should hydrate properties, and make them configurable', () => {
```

Comment:
This test doesn't cover the `catch` part of `hydrateObj` function. Would it be possible to improve it to simulate this `try`/`catch` behavior?

Reply:
I'm not able even manually to enter in the `catch` part of the modification. I'm not 100% sure of the impact but I'd go with either with a simple `obj[key] = value` (without `try...catch`) or with a modified version of `Object.defineProperty` that has a setter. In this case, you could let this unit test as it is.

Reply:
Yep OK thanks @alecpl I think I got it now.

@GeoSot This is not a good example but you can manage to go in the `catch` part via the unit test by doing stuff like that:

```js
EventHandler.on(div1, 'click', event => {
    Object.defineProperty(event, 'delegateTarget', {
        configurable: true,
        get() {
           return event
        }
    })
})

EventHandler.trigger(div1, 'click', { delegateTarget: div2, originalTarget: null })
```

Reply:
> This is probably not a good example 

This is probably the best example/help you could give us 😉 
Thank you a lot @julien-deramond